### PR TITLE
Animation displays faster so won't be so annoying to the user.

### PR DESCRIPTION
**Pull request by Anduin2017**

### CODE REVIEW

File: src/app/Services/InitService.ts [hunk 1]
```diff
@@ -114,7 +114,8 @@ export class InitService {
         this.messageService.resetVariables();
         this.cacheService.reset();
         this.messageService.me = null;
-        localStorage.clear();
+        // Don't clear it to save theme settings.
```
Comment (xxyzz) [inline src/app/Services/InitService.ts]: If someone signs in another account, he will use the previous one's setting and may try to resend some messages.

File: src/app/Styles/talking.css [hunk 2]
```diff
@@ -248,6 +248,7 @@
 
 .chat-part .chatinput-extented {
     width: 100%;
+    border-radius: 6px 6px 6px 6px;
```
Comment (xxyzz) [inline src/app/Styles/talking.css]: `border-radius: 6px;` should have the same effect.

File: src/app/Services/InitService.ts (hunk 1 above)
Comment (Anduin2017) [reply src/app/Services/InitService.ts]: So, what about save the preferred theme settings to our server side?

File: src/app/Styles/talking.css (hunk 2 above)
Comment (Anduin2017) [reply src/app/Styles/talking.css]: Good idea! Will fix.

File: src/app/Services/InitService.ts (hunk 1 above)
Comment (xxyzz) [reply src/app/Services/InitService.ts]: OK.

**Comment by Anduin2017** [comment]
@xxyzz @hv0905  Please review this again. Thanks!

File: src/app/Services/InitService.ts [hunk 3]
```diff
@@ -114,7 +114,6 @@ export class InitService {
         this.messageService.resetVariables();
         this.cacheService.reset();
         this.messageService.me = null;
-        localStorage.clear();
```
Comment (xxyzz) [inline src/app/Services/InitService.ts]: This line should not be deleted...I think

**Review by xxyzz — APPROVED** [review]

**Comment by Anduin2017** [comment]
What about waiting for @hv0905 's review?

**Pull request merged**